src/main/kotlin/org/malachite/estella/ApplicationProperties.kt

```diff
@@ -0,0 +1,57 @@
+package org.malachite.estella
+
+import com.beust.klaxon.Klaxon
+import java.io.File
+
+const val applicationPath = "src/main/resources/application.properties"
+const val configPath = "config2.json"
+
+fun getConfigurationData(): MutableMap<String, String> {
+    val file = File(configPath)
+    if (!file.exists())
+        return HashMap()
+    return Klaxon().parse<MutableMap<String, String>>(File(configPath)) ?: HashMap<String, String>()
```

Comment:
Wydaje mi się, że mamy już JacksonObjectMapper, co pewnie robi to samo - sprawdzisz, żeby się potem nie okazało, że faktycznie mamy 30 różnych bibliotek, które to samo robią?

Reply:
To jest od samego początku jak tworzyłem obsługę application.properties XD